src/cards/summon_decoy.ts

```diff
@@ -62,11 +62,6 @@ const spell: Spell = {
           // Animate effect of unit spawning from the sky
           skyBeam(unit);
         }
-        if (unit.image) {
-          const quantityScaleModifier = 1 + 0.3 * (quantity - 1);
-          unit.image.sprite.scale.x = unit.image.sprite.scale.x * quantityScaleModifier;
-          unit.image.sprite.scale.y = unit.image.sprite.scale.y * quantityScaleModifier;
-        }
```

Comment:
Is this already handled by the other summon logic?

Reply:
This is handled in unit.create now, I think.

I'd have to check, but yes it's handled elsewhere